README.md

```diff
@@ -18,6 +18,19 @@ Preview the template here: https://digital-garden-jekyll-template.netlify.app/
 
 <img width="1522" alt="Screen Shot 2020-05-19 at 23 05 46" src="https://user-images.githubusercontent.com/8457808/82400515-7d026d80-9a25-11ea-83f1-3b9cb8347e07.png">
 
+
+## Modifying for use with static output
+
+If you are using a site such as Neocities which cannot resolve urls that do not end with .html the bidirectional links will not work. This can be fixed by modiying ``` _plugins/bidirectional_links_generator.rb``` on lines ```5, 32, 39, 46 and 72``` to have the .html extension. 
```

Comment:
A typo fix and some formatting suggestions:

```suggestion
If you are using a site such as Neocities which cannot resolve URLs that do not end with `.html`, the bidirectional links will not work. This can be fixed by modifying ``` _plugins/bidirectional_links_generator.rb``` on lines ```5, 32, 39, 46 and 72``` to have the `.html` extension. 
```